framer/Layer.coffee

```diff
@@ -639,6 +639,12 @@ class exports.Layer extends BaseClass
 	##############################################################
 	## IMAGE
 
+	cleanupImageLoader: ->
```

Comment:
Fixed in https://github.com/nvh/Framer/commit/5c0fe7e5e78fa4889a8149a382311af5b83e88ba